samples/csharp/sharedcontent/console/Program.cs

```diff
@@ -155,10 +152,15 @@ static void Main(string[] args)
                     case ConsoleKey.V:
                         TranslationSamples.TranslationWithFileCompressedInputAsync().Wait();
                         break;
-
                     case ConsoleKey.W:
                         SpeechRecognitionSamples.KeywordRecognizer().Wait();
                         break;
+                    case ConsoleKey.X:
+                        SpeechSynthesisSamples.SynthesisWithAutoDetectSourceLanguageAsync().Wait();
+                        break;
```

Comment:
make sure you run the sample tests.
there is TCL scripts that click these menu items, and changing or adding menu items can break the sample tests.

Reply:
oh is it :), I though this was added by mistake. lets see the build and decide